### ISSUE TITLE
Failing migration after an update

### ISSUE THREAD

**Issue by paricbat** [go-vikunja/vikunja#367]
### Description

Hey,
I updated Vikunja and now it can't start because of a failed migration, any idea what might be causing this?
```
vikunja-1  | 2024-12-03T19:47:20.598110689Z: INFO       ▶ [DATABASE] 06a [SQL] SELECT `id`, `bucket_configuration` FROM `project_views` WHERE (bucket_configuration_mode != 1 AND view_kind = 3) [] - 37.056µs
vikunja-1  | 2024-12-03T19:47:20Z: CRITICAL     ▶ 06b Migration failed: migration 20240919130957 failed: no columns found to be updated
```

### Vikunja Version

0.24.5

### Can you reproduce the bug on the Vikunja demo site?

No

**Comment by kolaente** [comment]
Does it work if you run the migration again?

**Comment by paricbat** [comment]
> Does it work if you run the migration again?

It does not

**Comment by Zomphie** [comment]
I am having the same issue. Though, some of the data does import, but not all of it and the little it does, it doesn't keep its directory format.

**Comment by kolaente** [comment]
This particular problem should be fixed in 942c2e4af6eb6040e287d34d92061a40c3dbffa3, please check with the next unstable build (should be ready for deployment in ~45min, also on [try](https://try.vikunja.io)).
